lib/check_params.h

```diff
@@ -342,6 +342,8 @@ void printQudaInvertParam(QudaInvertParam *param) {
   P(m5, INVALID_DOUBLE);
   P(Ls, INVALID_INT);
   P(mu, INVALID_DOUBLE);
+  P(epsilon, INVALID_DOUBLE);
```

Comment:
this is unrelated but needed right?

Reply:
It was somehow never added. None of these are really needed, but it makes a cleaner parameter struct if it can be properly initialised with default values and the checks will catch incorrect entries where possible.